guideline.md

```diff
@@ -0,0 +1,126 @@
+# Pyladies Germany PySV Grant Program
+
+The annual €5000 grant program from the PySV for PyLadies aims to support the activities and growth of PyLadies chapters in Germany. It is essential to ensure that these funds are used transparently, therefore we have devised the following guidelines. The grant will be issued to PyLadies each year after the PyCon DE conference (circa June), once assigned the funds are available for disposal by the PyLadies using the following guidelines. Money not used within the year will stay on the PyLadies account and can continue to be used.
+
+**Allowable expenses:**
+- Community events: this includes room rentals, refreshments, and similar costs.
```

Comment:
should we add devices to be used on meetups events: microphone, camera, lightning?